### DISCUSSION TITLE
404 Not Found (error code: 10062): Unknown interaction

### DISCUSSION THREAD

**Discussion by EvokerKing** [nextcord/nextcord#807] [Q&A] [answered]
Code:
```py
summarized = summarizer(text, max_length=maxLength, min_length=minLength, do_sample=False)
summarized = summarized[0]["summary_text"]
await ctx.send(f"Summarized: \"{summarized}\"")
```
When I use the command it sits there for a moment and then says "The application did not respond" but nothing happens. Then after about 15 seconds, it returns this error:
```
nextcord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction

The above exception was the direct cause of the following exception:

nextcord.errors.ApplicationInvokeError: Command raised an exception: NotFound: 404 Not Found (error code: 10062): Unknown interaction
```

I can't find anything about this anywhere except a deleted Reddit post that I can't use for help since the comments that explain the solution are also deleted.

**Comment by ooliver1** [comment]
We need more info, what does `summarizer` do?

**Reply by Skelmis** [reply]
Also what is `ctx` defined as?

**Reply by EvokerKing** [reply]
Summarizer is a function for summarizing a string of text. I am still used to Discord.py so I use CTX instead of interaction.

**Comment by DenverCoder1** [comment, marked as answer]
Unknown interaction usually means you are taking too long to respond to an interaction.

Discord expects you to respond in 3 seconds, you can extend that by deferring and then sending followup in the next 15 minutes.

Responding to the interaction with a [defer](https://docs.nextcord.dev/en/latest/api.html#nextcord.InteractionResponse.defer) response will show the “bot is thinking” message to the user until the followup response is sent.

Note: by default the message will be public, but you can make it visible only to the user by setting the ephemeral parameter to True.

See also:

https://docs.nextcord.dev/en/latest/interactions.html#deferred-response

**Reply by EvokerKing** [reply]
```py
await nextcord.InteractionResponse.defer(ctx, ephemeral=True, with_message=True)
```
I tried this but now it returns:
```
AttributeError: 'Interaction' object has no attribute '_responded'

The above exception was the direct cause of the following exception:

nextcord.errors.ApplicationInvokeError: Command raised an exception: AttributeError: 'Interaction' object has no attribute '_responded'
```

**Reply by EvokerKing** [reply]
Fixed it by creating a variable class with `ctx.response`
```py
interactionResponse = ctx.response
await interactionResponse.defer(ephemeral=True, with_message=True)
```
